src/Commands/MakeOverride.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * Copyright (c) Since 2020 Friends of Presta
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file docs/licenses/LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to infos@friendsofpresta.org so we can send you a copy immediately.
+ *
+ * @author    Friends of Presta <infos@friendsofpresta.org>
+ * @copyright since 2020 Friends of Presta
+ * @license   https://opensource.org/licenses/AFL-3.0  Academic Free License ("AFL") v. 3.0
+ */
+declare(strict_types=1);
+
+namespace FOP\Console\Commands;
+
+use Exception;
+use FOP\Console\Command;
+use FOP\Console\Overriders\OverriderInterface;
+use FOP\Console\Overriders\Provider;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class MakeOverride extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setName('fop:override')
```

Comment:
La commande doit respecter la convention domain:action ?
Du coup ici pq pas fop:override:make

Reply:
effectivement, on pourrait faire ça, bien vu.
C'est noté.
la commande `fop:override` continuera de fonctionner (puisque c'est la seule dans le domaine override).
J'en ai discuté avec @jf-viguier et j'avais dit le contraire mais finalement, je pense que c'est mieux de s'en tenir à la convention (et ça évitera une modification si on a une commande pour désactiver ou supprimer un override plus tard).